src/fides/ctl/core/config/database_settings.py

```diff
@@ -26,7 +26,7 @@ class DatabaseSettings(FideslibDatabaseSettings):
     test_db: str = "default_test_db"
 
     async_database_uri: Optional[PostgresDsn]
-    sync_database_uri: Optional[PostgresDsn]
+    sync_database_uri: Optional[str]
```

Comment:
Should we also update `async_database_uri` to be `Optional[str]`, for consistency?